doc/developers/contributing.rst

```diff
@@ -14,6 +14,23 @@ adding new algorithms, and the best way to contribute and to help the project
 is to start working on known issues.
 See :ref:`easy_issues` to get started.
 
+.. topic:: **Our community, our values**
+
+    We are a community based on openness and friendly discussions.
+
+    We aspire to treat everybody equal, and value their contributions.
+
+    Decisions are made based on technical merit and consensus.
+
+    Code is not the only way to help the project. Answering questions to
+    help others on mailing lists or issues, organizing and teaching
+    tutorials, working on the website, are all priceless contributions.
+
+    We abide by the principles of openness and respect of the Python
+    Software Fundation: https://www.python.org/psf/codeofconduct/
```

Comment:
> Typo in Foundation.

Thanks

> Any particular reason that the principle of "considerate" was left out or were
> you thinking that openness and respect captured it?

That was my thinking, but you are right that "explicit is better than
implicit", so I'll put it back in, in particular since such consideration
is very important to me.